easy_login/tests/tests.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.test import RequestFactory, TestCase, Client
+from bs4 import BeautifulSoup
+from easy_login.context_processors import easy_login
+
+import re
+
+
+class EasyLoginTest(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
```

Comment:
please try not to use self variables in setup.
it's lead to new bugs in your test.
better define this variable in separate script.